client/src/components/backend/Form/MaskedTextInput.js

```diff
@@ -13,7 +13,11 @@ export default class FormMaskedTextInput extends Component {
 
   static propTypes = {
     ...sharedPropsValidation,
-    mask: PropTypes.string.isRequired
+    mask: PropTypes.oneOfType(
```

Comment:
This is wrong. OneOfType takes one argument, which is an array of allowed types. You're passing it multiple arguments.